content/blog/2020/Weeknotes3.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Week3-20200607"
+date: 2020-06-15T19:08:09-04:00
+categories: ["RIT","Weeknotes","Web Development"]
+---
+As mentioned in the previous week, I learned the basics of sqlite database using the website and simultaneously created the basic todo webapp using the Youtube tutorial.
+### **SQLite learning process**
+The website uses a good strategy **(Spaced Repetition system)** to teach the required content and only 16 lessons are free. So it takes the whole week to progress through the free lessons.
+>See previous weeknote for the site's link
+### **Basic todo webapp**
+These are the steps used in the tutorial.  
```

Comment:
Give a brief description of this project. Say that it is a flask web app with an SQLite database. 

Say a few words about the features of the application you've created. 

If this is deployed then include a link to the app so the users can try it out themselves. 

Remember "Kanden Seethaiyai" attitude when you're writing the blog post. Start with the most important things.